[quant][graphmode[fx][test][refactor] Refactor tests for graph mode quantization on fx

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43526 [quant][graphmode][fx] Testing torchvision
* **#43445 [quant][graphmode[fx][test][refactor] Refactor tests for graph mode quantization on fx**

Summary:
changed the interface for checkGraphModule to make the arguments more explicit
as requested in https://github.com/pytorch/pytorch/pull/43437

Test Plan:
TestQuantizeFx

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23280586](https://our.internmc.facebook.com/intern/diff/D23280586)

jerryzh168 committed:
[quant][graphmode[fx][test][refactor] Refactor tests for graph mode quantization on fx

Summary:
changed the interface for checkGraphModule to make the arguments more explicit
as requested in https://github.com/pytorch/pytorch/pull/43437

Test Plan:
TestQuantizeFx

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode[fx][test][refactor] Refactor tests for graph mode quantization on fx"


Summary:
changed the interface for checkGraphModule to make the arguments more explicit
as requested in https://github.com/pytorch/pytorch/pull/43437

Test Plan:
TestQuantizeFx

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23280586](https://our.internmc.facebook.com/intern/diff/D23280586)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode[fx][test][refactor] Refactor tests for graph mode quantization on fx"


Summary:
changed the interface for checkGraphModule to make the arguments more explicit
as requested in https://github.com/pytorch/pytorch/pull/43437

Test Plan:
TestQuantizeFx

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23280586](https://our.internmc.facebook.com/intern/diff/D23280586)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode[fx][test][refactor] Refactor tests for graph mode quantization on fx"


Summary:
changed the interface for checkGraphModule to make the arguments more explicit
as requested in https://github.com/pytorch/pytorch/pull/43437

Test Plan:
TestQuantizeFx

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23280586](https://our.internmc.facebook.com/intern/diff/D23280586)

[ghstack-poisoned]

vkuzo commented on torch/testing/_internal/common_quantization.py:
```diff
@@ -450,26 +488,22 @@ def printGraphModule(self, graph_module, print_str=True):
         """ Quantizes model with graph mode quantization on fx and check if the
         quantized model contains the quantized_node
 
         Args:
```
optional: since the argument types are non-trivial, would be great to include a short code sample (so code reader can copy-paste from here instead of looking at callsites, reading NodeSpec docs, etc.

jerryzh168 replied:
Sure

vkuzo reviewed (approved):
thanks for improving this!

jerryzh168 committed:
Update on "[quant][graphmode[fx][test][refactor] Refactor tests for graph mode quantization on fx"


Summary:
changed the interface for checkGraphModule to make the arguments more explicit
as requested in https://github.com/pytorch/pytorch/pull/43437

Test Plan:
TestQuantizeFx

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23280586](https://our.internmc.facebook.com/intern/diff/D23280586)

[ghstack-poisoned]
